capten/agent/pkg/capten-store/configure_projects.go

```diff
@@ -0,0 +1,126 @@
+package captenstore
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/gocql/gocql"
+	"github.com/kube-tarian/kad/capten/agent/pkg/model"
+	"github.com/pkg/errors"
+)
+
+const (
+	TektonTableName = "TektonProjects"
+)
+
+const (
+	getConfigProjectsQuery      = "SELECT id, git_project_id, status, last_update_time, workflow_id FROM %s.%s;"
+	getConfigProjectsForIDQuery = "SELECT id, git_project_id, status, last_update_time, workflow_id FROM %s.%s WHERE id=%s;"
+	insertConfigProjectQuery    = "INSERT INTO %s.%s(id, git_project_id, status, last_update_time, workflow_id) VALUES (?,?,?,?,?);"
+	updateConfigProjectQuery    = "UPDATE %s.%s SET status='%s', last_update_time='%s', workflow_id='%s' WHERE id=%s;"
+	deleteConfigProjectQuery    = "DELETE FROM %s.%s WHERE id=%s;"
+)
+
+func (a *Store) UpsertConfigProject(project *model.ConfigureProject, tablename string) error {
+	project.LastUpdateTime = time.Now().Format(time.RFC3339)
```

Comment:
Don't generalize DB table handling.
only workflow and git handling need generic implementations